--- a/src/vldp/vldp.h
+++ b/src/vldp/vldp.h
@@ -35,6 +35,8 @@
 // But CMake does the right thing here 
 #include <SDL.h> // only used for threading
 
+#include <mpeg2dec/mpeg2.h>
+
 struct yuv_buf {
     uint8_t *Y;     // Y channel
     uint8_t *U;     // U channel
--- a/src/vldp/vldp_internal.h
+++ b/src/vldp/vldp_internal.h
@@ -27,8 +27,6 @@
 
 #include "vldp.h" // for the VLDP_BOOL definition and SDL.h
 
-#include <mpeg2.h>
-
 // this is which version of the .dat file format we are using
 #define DAT_VERSION 3
 
