--- a/src/gameconfigfile.cpp	2026-10-01 18:35:29.044955674 +0200
+++ b/src/gameconfigfile.cpp	2026-10-01 18:36:15.899459754 +0200
@@ -150,7 +150,7 @@
 		SetValueForKey ("Path", SHARE_DIR "/games/doom", true);
 		SetValueForKey ("Path", "/usr/local/share/doom", true);
 		SetValueForKey ("Path", "/usr/local/share/games/doom", true);
-		SetValueForKey ("Path", "/usr/share/doom", true);
+		SetValueForKey ("Path", "/usr/share/gzdoom", true);
 		SetValueForKey ("Path", "/usr/share/games/doom", true);
 #endif
 		SetValueForKey ("Path", "$DOOMWADDIR", true);
@@ -181,8 +181,8 @@
 		SetValueForKey("Path", "/usr/local/share/doom/fm_banks", true);
 		SetValueForKey("Path", "/usr/local/share/games/doom/soundfonts", true);
 		SetValueForKey("Path", "/usr/local/share/games/doom/fm_banks", true);
-		SetValueForKey("Path", "/usr/share/doom/soundfonts", true);
-		SetValueForKey("Path", "/usr/share/doom/fm_banks", true);
+		SetValueForKey("Path", "/usr/share/gzdoom/soundfonts", true);
+		SetValueForKey("Path", "/usr/share/gzdoom/fm_banks", true);
 		SetValueForKey("Path", "/usr/share/games/doom/soundfonts", true);
 		SetValueForKey("Path", "/usr/share/games/doom/fm_banks", true);
 		SetValueForKey("Path", SHARE_DIR "/doom/soundfonts", true);
--- a/src/common/platform/posix/sdl/i_main.cpp	2026-10-01 18:35:29.331944598 +0200
+++ b/src/common/platform/posix/sdl/i_main.cpp	2026-10-01 18:36:15.899583282 +0200
@@ -176,23 +176,7 @@
 
 	Args = new FArgs(argc, argv);
 
-#ifdef PROGDIR
-	progdir = PROGDIR;
-#else
-	char program[PATH_MAX];
-	if (realpath (argv[0], program) == NULL)
-		strcpy (program, argv[0]);
-	char *slash = strrchr (program, '/');
-	if (slash != NULL)
-	{
-		*(slash + 1) = '\0';
-		progdir = program;
-	}
-	else
-	{
-		progdir = "./";
-	}
-#endif
+progdir = "/usr/share/gzdoom/";
 
 	I_StartupJoysticks();
 
--- a/src/common/platform/posix/unix/i_specialpaths.cpp	2026-10-01 18:35:29.618933521 +0200
+++ b/src/common/platform/posix/unix/i_specialpaths.cpp	2026-10-01 18:36:15.899531937 +0200
@@ -132,7 +132,7 @@
 {
 	// Don't use GAME_DIR and such so that ZDoom and its child ports can
 	// share the node cache.
-	FString path = NicePath("$HOME/.config/zdoom/cache");
+	FString path = NicePath("$HOME/.config/gzdoom/cache");
 	if (create)
 	{
 		CreatePath(path.GetChars());
